packages/core/src/lib/connection_manager.ts

```diff
@@ -25,10 +23,7 @@ export const DEFAULT_MAX_BOOTSTRAP_PEERS_ALLOWED = 1;
 export const DEFAULT_MAX_DIAL_ATTEMPTS_FOR_PEER = 3;
 export const DEFAULT_MAX_PARALLEL_DIALS = 3;
 
-export class ConnectionManager
-  extends TypedEventEmitter<IPeersByDiscoveryEvents & IConnectionStateEvents>
-  implements IConnectionManager
-{
+export class ConnectionManager implements IConnectionManager {
```

Comment:
would like to discuss if you think it is reasonable to leverage `libp2p` for all events we have

@waku-org/js-waku-developers

Reply:
I mean this change https://github.com/waku-org/js-waku/pull/2049/commits/d17cd913aa280e67de58a88de10956250754abbb

Reply:
dropping this out into https://github.com/waku-org/js-waku/issues/2067